content/en/network_monitoring/performance/network_analytics.md

```diff
@@ -21,7 +21,11 @@ further_reading:
       text: 'Collect your Network Data with the Datadog Agent.'
 ---
 
-{{< img src="network_performance_monitoring/network_analytics/main_page_npm2.png" alt="Main page" >}}
+## Overview
+
+The Network Analytics page provides insights into your overall network health and shows [recommended queries](#recommended-queries) at the top of the page. These recommended queries provide simplified access to run common queries, and provide a snapshot of related metrics to the query so that you can see relevant changes in throughput, latency, DNS errors, and more. Clicking on a recommended query automatically populates the search bar, group bys, and summary graphs into a unique combination to provide you with the most relevant insights into your network.
```

Comment:
```suggestion
The Network Analytics page provides insights into your overall network health and shows [recommended queries](#recommended-queries) at the top of the page. These recommended queries enable you to run common queries and see snapshots of relevant metrics, so that you can see changes in throughput, latency, DNS errors, and more. Clicking on a recommended query automatically populates the search bar, group bys, and summary graphs to provide you with relevant insights into your network.
```

Rewording some things:
- I think "provide simplified access to run" is more verbose than it needs to be. Also, "simplified" tends to be a bit subjective—it isn't especially egregious here, but I like to take it out whenever possible.
- smoothed out the parallel construction a bit
- "related metrics to the query" -> "relevant metrics"
- wasn't sure about "a unique combination" (is it truly unique in the literal sense? do users particularly value uniqueness in this context?) but it seems unnecessary, removing it
- "most relevant" is subjective